vmdb/app/assets/javascripts/miq_application.js

```diff
@@ -58,93 +84,127 @@ function miqPrepRightCellForm(tree) {
 
 // Things to be done on page resize
 function miqOnResize() {
-//  if ($('miq_timeline')) miqResizeTL();
-  if (typeof dhxLayout != "undefined") dhxLayout.setSizes();
+  if (typeof dhxLayout != "undefined") {
+    dhxLayout.setSizes();
+  }
+
   miqGetBrowserSize();
 }
 
 // Initialize the widget pulldown on the dashboard
 function miqInitWidgetPulldown() {
   var miqMenu = new dhtmlXToolbarObject("widget_select_div", "miq_blue");
+
   miqMenu.setIconsPath("/images/icons/24/");
   miqMenu.attachEvent("onClick", miqWidgetToolbarClick);
   miqMenu.loadXMLString(miqMenuXML); // miqMenuXML var is loaded in dashboard/_dropdownbar.rhtml
   miqSetToolbarButtonIds(miqMenu);
 }
 
-function miqCalendarDateConversion(server_offset){
-  x=new Date(tN().getUTCFullYear(),tN().getUTCMonth(),tN().getUTCDate(),tN().getUTCHours(),tN().getUTCMinutes(),tN().getUTCSeconds());if(server_offset >= 0){x.setTime(x.getTime()+(Number(server_offset))*1000)}else{x.setTime(x.getTime()-(Number(server_offset.slice(1,server_offset.length)))*1000)};
+function miqCalendarDateConversion(server_offset) {
+  var tn = new Date();
+  var x = new Date(
+    tn.getUTCFullYear(),
+    tn.getUTCMonth(),
+    tn.getUTCDate(),
+    tn.getUTCHours(),
+    tn.getUTCMinutes(),
+    tn.getUTCSeconds()
+  );
+
+  if (server_offset >= 0) {
+    x.setTime(x.getTime() + (Number(server_offset)) * 1000);
+  } else {
+    x.setTime(x.getTime() - (Number(server_offset.slice(1, server_offset.length))) * 1000);
+  }
+
   return x;
 }
 
 // Track the mouse coordinates for popup menus
-var miqMouseX, miqMouseY;
-function miqGetMouseXY(positionX, positionY){
-  miqMouseX = positionX,
+var miqMouseX;
+var miqMouseY;
+
+function miqGetMouseXY(positionX, positionY) {
+  miqMouseX = positionX;
   miqMouseY = positionY;
 }
 
 // Prefill text entry field when blank
 function miqLoginPrefill() {
-  if (miqDomElementExists('user_name')) miqPrefill($('#user_name')[0]);
-  if (miqDomElementExists('user_password')) miqPrefill($('#user_password')[0]);
-  if (miqDomElementExists('user_new_password')) miqPrefill($('#user_new_password')[0]);
-  if (miqDomElementExists('user_verify_password')) miqPrefill($('#user_verify_password')[0]);
-  if (miqDomElementExists('user_name')) self.setTimeout('miqLoginPrefill()',200); // Retry in .2 seconds, if user name field is present
+  if (miqDomElementExists('user_name')) {
+    miqPrefill($('#user_name')[0]);
```

Comment:
in another PR, just don't expand the scope of this to rewrite all the JS at once